fineract-provider/src/main/java/org/apache/fineract/accounting/journalentry/service/AccountingProcessorHelper.java

```diff
@@ -125,6 +125,7 @@ public LoanDTO populateLoanDtoFromMap(final Map<String, Object> accountingBridge
             final BigDecimal overPayments = (BigDecimal) map.get("overPaymentPortion");
             final boolean reversed = (Boolean) map.get("reversed");
             final Long paymentTypeId = (Long) map.get("paymentTypeId");
+            boolean isDisburseToSavings = (Boolean) map.get("isDisburseToSavings");
```

Comment:
use get or default , or ell you might get a NULL pointer exception

Reply:
I added that particular field from both places from where we are using it.

Reply:
final boolean reversed = (Boolean) map.getOrDefault("reversed", false);
boolean isDisburseToSavings = (boolean) map.getOrDefault("isDisburseToSavings", false);